### PR TITLE
SONARGRADL-60 Set the property "sonar.coverage.jacoco.xmlReportPaths"…

### PR DESCRIPTION
**Pull request by saberduck**
… when the "Gradle Jacoco Report" is configured

### CODE REVIEW

File: src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java [hunk 1]
```diff
@@ -243,6 +245,13 @@ private static void extractTestProperties(Project project, Map<String, Object> p
   }
 
   private static void configureJaCoCoCoverageReport(final Test testTask, final boolean addForGroovy, Project project, final Map<String, Object> properties) {
+    project.getTasks().withType(JacocoReport.class, jacocoReportTask -> {
+      SingleFileReport xmlReport = jacocoReportTask.getReports().getXml();
+      if (xmlReport.isEnabled() && xmlReport.getDestination().exists()) {
```
Comment (henryju) [inline src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java]: Maybe we could log a message when xml report is not enabled?

**Comment by henryju** [comment]
@saberduck Any progress on this?

**Comment by saberduck** [comment]
I prepared PR on sonar-enterprise https://github.com/SonarSource/sonar-enterprise/pull/1526 , the logs look ok https://api.cirrus-ci.com/v1/task/5100667390656512/logs/main.log

File: src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java (hunk 1 above)
Comment (saberduck) [reply src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java]: log added

**Comment by salvianreynaldi** [comment]
Hi @saberduck,
How do I set `sonar.coverage.jacoco.xmlReportPaths`?
I tried using version 2.7.1 of this plugin (which should include this PR). When I replace 
`sonar.jacoco.reportPaths` with `sonar.coverage.jacoco.xmlReportPaths` in the root project of a multi-project build, and gradle complained that the new property is unknown (still the same as in version 2.7). I also tried moving the property setting from root to the leaf projects, and the error persists
Thank you

**Comment by saberduck** [comment]
hello @salvianreynaldi , 
can you please open topic on https://community.sonarsource.com/ and provide the details from your build? How are you trying to set the property?
This PR is merged & closed, so I would prefer not to continue discussion here.

**Comment by salvianreynaldi** [comment]
Hi,
please ignore my report. I set the parameter in a comma-delimited string ("path1,path2") as said in [JaCoCo Plugin Docs](https://docs.sonarqube.org/display/PLUG/JaCoCo+Plugin). It turns out you have to use list of strings (just like how `sonar.jacoco.reportPaths` is set) if you set it in gradle (["path1", "path2"]). Thanks

**Pull request merged**